src/replication_handler/replication_handler.cpp

```diff
@@ -271,9 +271,8 @@ auto ReplicationHandler::GetDatabasesHistories() -> replication_coordination_glu
   dbms_handler_.ForEach([&results](memgraph::dbms::DatabaseAccess db_acc) {
     auto &repl_storage_state = db_acc->storage()->repl_storage_state_;
 
-    std::vector<std::pair<std::string, uint64_t>> history =
-        utils::fmap([](const auto &elem) { return std::pair<std::string, uint64_t>(elem.first, elem.second); },
-                    repl_storage_state.history);
+    std::vector<std::pair<std::string, uint64_t>> history = utils::fmap(
+        [](const auto &elem) { return std::make_pair(elem.first, elem.second); }, repl_storage_state.history);
```

Comment:
FYI:
1. no need for `std::make_pair` now we have CTAD, `std::pair{elem.first, elem.second}`
2. no need for lambda, pair -> pair, fixup `fmap` with correct result `typename R = std::decay_t<std::invoke_result_t<F, T>>` then `std::identity{}` can be used.
3. no need for fmap, `auto history = std::vector(repl_storage_state.history.cbegin(), repl_storage_state.history.cend());`